### PR TITLE
xbps-src: portability fixes

### PR DESCRIPTION
**Pull request by Duncaen**

### CODE REVIEW

File: xbps-src [hunk 1]
```diff
@@ -415,7 +415,7 @@ XBPS_OPTIONS=
 XBPS_OPTSTRING="a:CEfgGhH:Ij:Lm:No:r:tV"
 
 # Preprocess arguments in order to allow options before and after XBPS_TARGET.
-eval set -- $(getopt -- "$XBPS_OPTSTRING" "$@");
+eval set -- $(getopt "$XBPS_OPTSTRING" "$@");
```
Comment (Gottox) [inline xbps-src]: What does this fix?

Comment (Duncaen) [inline xbps-src]: My `getopt` does not support it, see the `COMPATIBILITY` section of GNU `getopt`.

Comment (Gottox) [inline xbps-src]: ok, thx :)

**Comment by Vaelatern** [comment]
:+1:

**Pull request merged**